Source/WebCore/platform/graphics/gstreamer/mse/WebKitMediaSourceGStreamer.cpp

```diff
@@ -255,6 +255,13 @@ static const char* streamTypeToString(TrackPrivateBaseGStreamer::TrackType type)
 
 static gboolean webKitMediaSrcQuery(GstElement* element, GstQuery* query)
 {
+#if GST_CHECK_VERSION(1, 24, 9)
```

Comment:
The regression you are fixing is for a runtime change in GStreamer 1.24.9. If you want to make this explicit, then add a `webkitGstCheckVersion(1, 24, 9)`, but that's not really needed imho.

I was asking for an ifdef for this query type that was first shipped in 1.22.0.

Reply:
Ok my bad, I will fix this

Reply:
> I was asking for an ifdef for this query type that was first shipped in 1.22.0.

You mean I just need to change the version to 1.22.0 in the check_version ? else I did not get what you mean by ifdef

Reply:
Yes, only that :)